package.json

```
@@ -3,18 +3,18 @@
   "version": "1.0.0",
   "devDependencies": {
     "cross-env": "^5.1.5",
-    "css-loader": "^0.28.11",
+    "css-loader": "^1.0.0",
     "mini-css-extract-plugin": "^0.4.0",
-    "serve": "^6.5.5",
-    "style-loader": "^0.21.0",
+    "serve": "^10.0.2",
+    "style-loader": "^0.23.1",
     "svelte": "^2.0.0",
-    "svelte-loader": "2.9.0",
+    "svelte-loader": "2.11.0",
     "webpack": "^4.8.3",
-    "webpack-cli": "^2.0.14",
-    "webpack-serve": "^1.0.2"
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.9"
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack",
-    "dev": "webpack-serve --content public --hot"
+    "dev": "webpack-dev-server"
```

```suggestion
    "dev": "webpack-dev-server --content-base public"
```
That works for me. Original command simply show a list of files.

---

Yes. That's correct, thanks!